src/Compiler/Service/ServiceStructure.fs

```diff
@@ -364,30 +364,30 @@ module Structure =
                                members = ms
                                extraImpls = extraImpls
                                newExprRange = newRange
-                               range = wholeRange) ->
+                               range = mWhole) ->
                 let bindings = unionBindingAndMembers bindings ms
 
                 match argOpt with
                 | Some (args, _) ->
-                    let collapse = Range.endToEnd args.Range wholeRange
-                    rcheck Scope.ObjExpr Collapse.Below wholeRange collapse
+                    let collapse = Range.endToEnd args.Range mWhole
+                    rcheck Scope.ObjExpr Collapse.Below mWhole collapse
                 | None ->
-                    let collapse = Range.endToEnd newRange wholeRange
-                    rcheck Scope.ObjExpr Collapse.Below wholeRange collapse
+                    let collapse = Range.endToEnd newRange mWhole
+                    rcheck Scope.ObjExpr Collapse.Below mWhole collapse
 
                 parseBindings bindings
                 parseExprInterfaces extraImpls
 
-            | SynExpr.TryWith (e, matchClauses, wholeRange, tryPoint, withPoint, _trivia) ->
+            | SynExpr.TryWith (e, matchClauses, mWhole, tryPoint, withPoint, _trivia) ->
```

Comment:
This particular one actually seems a bit less clear now, partly because the code below has some `fullrangeTry` which is likely related.

Reply:
`m` is a shorthand for `marker` which is used to represent a range in many places in the compiler. This is clear enough after you discover it :)

Reply:
There is a lot of irregularity between `m*` and `*Range`.  We'd probably better regularize to `m*` partly because I don't see us removing `m`.

In this particular case we had both `wholeRange` and `mWhole` in the codebase and I regularised to the latter.